Assets/Mirror/Runtime/NetworkManager.cs

```diff
@@ -315,11 +311,40 @@ public bool StartServer()
             }
             else
```

Comment:
you can just get rid of this else completely and expect the caller to call SpawnObjects at a later point